### PR TITLE
feat: Add auxiliary lines between list points

### PR DESCRIPTION
**Pull request by lihqi**

### CODE REVIEW

File: packages/lb-annotation/src/core/toolOperation/pointOperation.ts [hunk 1]
```diff
@@ -901,6 +901,59 @@ class PointOperation extends BasicToolOperation {
     }
   }
 
+  /**
+   * 渲染辅助线-需开启列表标注
```
Comment (Kerwin-L) [inline packages/lb-annotation/src/core/toolOperation/pointOperation.ts:905]: Please comment by English or Chinese & English

File: packages/lb-annotation/src/core/toolOperation/pointOperation.ts [hunk 2]
```diff
@@ -901,6 +901,59 @@ class PointOperation extends BasicToolOperation {
...
+    }
+
+    const auxiliaryLines = MarkerUtils.getAuxiliaryLineByMarkerList(this.config.markerList, 'value');
+    const auxiliaryLinesCoord = MarkerUtils.getAuxiliaryLineCoord(auxiliaryLines, pointList);
+
+    auxiliaryLinesCoord.forEach((item) => {
+      const { start: startCoord, end: endCoord } = item;
+      DrawUtils.drawLine(
+        this.canvas,
+        AxisUtils.changePointByZoom(
```
Comment (Kerwin-L) [inline packages/lb-annotation/src/core/toolOperation/pointOperation.ts:936]: You can use AxisUtils.changePointListByZoom.

File: packages/lb-annotation/src/utils/tool/MarkerUtils.ts [hunk 3]
```diff
@@ -1,5 +1,16 @@
+import { cloneDeep } from 'lodash';
 import AttributeUtils from './AttributeUtils';
 
+declare interface IAuxiliaryLine {
+  start: string;
+  end: string;
+}
+
+declare interface IAuxiliaryLineCoord {
```
Comment (Kerwin-L) [inline packages/lb-annotation/src/utils/tool/MarkerUtils.ts:9]: What is difference between IAuxiliaryLine and IAuxiliaryLineCoord？

File: packages/lb-annotation/src/utils/tool/MarkerUtils.ts [hunk 4]
```diff
@@ -10,4 +21,66 @@ export default class MarkerUtils {
...
+
+  /**
+   * 根据辅助线和点数据生成辅助线首尾坐标
+   */
+  static getAuxiliaryLineCoord = (auxiliaryLines: IAuxiliaryLine[], pointList: IPointUnit[]): IAuxiliaryLineCoord[] => {
+    const result: IAuxiliaryLineCoord[] = [];
+
+    auxiliaryLines.forEach((line) => {
+      const { start, end } = line;
+      const startPoint = pointList.find((item) => item.label === start);
```
Comment (Kerwin-L) [inline packages/lb-annotation/src/utils/tool/MarkerUtils.ts:61]: Why is label used? I think it is more correct to use attribute

Comment (lihqi) [reply packages/lb-annotation/src/utils/tool/MarkerUtils.ts:61]: markerList use `label`
attributeList use `attribute`

File: packages/lb-annotation/src/utils/tool/MarkerUtils.ts (hunk 3 above)
Comment (lihqi) [reply packages/lb-annotation/src/utils/tool/MarkerUtils.ts:9]: I want to change `IAuxiliaryLine` this way
```typescript
declare interface IAuxiliaryLine {
  start: IPointUnit['label'];
  end: IPointUnit['label'];
}
```

File: packages/lb-annotation/src/utils/tool/MarkerUtils.ts (hunk 4 above)
Comment (Kerwin-L) [reply packages/lb-annotation/src/utils/tool/MarkerUtils.ts:61]: Yes, you are right!

File: packages/lb-annotation/src/utils/tool/MarkerUtils.ts (hunk 3 above)
Comment (Kerwin-L) [reply packages/lb-annotation/src/utils/tool/MarkerUtils.ts:9]: Thanks for your description~

**Pull request merged**